docs/topic_guide/channel_response.rst

```diff
@@ -0,0 +1,148 @@
+.. _sunkit-instruments-topic-guide-channel-response:
+
+************************************
+A Vocabulary for Instrument Response
+************************************
+
+This topic guide provides a vocabulary for response functions for imaging instruments.
+The goal of providing a single vocabulary for instrument response calculations is to provide a specification for a common interface that can be used by multiple instruments.
+This reduces the amount of effort needed to develop analysis software for new instruments and enables cross-instrument comparisons as upstream packages and users can program against a single interface for these response calculations.
+An abstract implementation of this vocabulary is provided in this package.
+
+Temperature Response
+--------------------
+
+The temperature response describes the instrument sensitivity as a function of temperature.
+It is a useful quantity when performing thermal analysis of imaging data, such as a differential emission measure or filter ratio analysis.
+The temperature response is defined as,
+
+.. math::
+
+    K(T) = \int\mathrm{d}\lambda\,R(\lambda)S(\lambda,T)\quad[\mathrm{DN}\,\mathrm{pixel}^{-1}\,\mathrm{s}^{-1} \,\mathrm{cm}^5]
+
+It has a physical type of data number (DN) per pixel per unit time per unit emission measure.
+Note that the temperature response is a function of *both* the instrument properties as well as the atomic physics of the emitting source.
+The temperature response is related to the observed intensity in a given pixel by,
+
+.. math::
+
+    I = \int\mathrm{d}T\,K(T)\mathrm{DEM}(T)\quad[\mathrm{DN}\,\mathrm{pixel}^{-1}\,\mathrm{s}^{-1}],
+
+where :math:`\mathrm{DEM}(T)=n^2\frac{dh}{dT}` is the line-of-sight *differential* emission measure distribution in a given pixel and is typically express in units of :math:`\mathrm{cm}^{-5}\,\mathrm{K}^{-1}`. 
+
+Source Spectra
+--------------
+
+The source spectra describes how a source is emitting as a function of wavelength and temperature.
+It is denoted by :math:`S(\lambda, T)`.
+The source spectra has a physical type of photon per unit time per unit wavelength per solid angle per unit density.
+The units are commonly expressed as
+:math:`\mathrm{photon}\,\mathrm{s}^{-1}\,\mathring{\mathrm{A}}^{-1}\,\mathrm{sr}^{-1}\,\mathrm{cm}^3`.
+The source spectra is specified by the user and can be computed from atomic databases (e.g. CHIANTI).
+This quantity is independent of any instrument properties.
+
+Wavelength Response
+-------------------
+
+The wavelength response describes the instrument sensitivity as a function of wavelength and time.
+The wavelength response is defined as,
+
+.. math::
+
+    R(\lambda,t) = A_{\mathrm{eff}}(\lambda,t)f(\lambda)\frac{pg}{s}\quad[\mathrm{cm}^2\,\mathrm{DN}\,\mathrm{sr}\,\mathrm{photon}^{-1}\,\mathrm{pixel}^{-1}]
+
+It has a physical type of area DN solid angle per photon per pixel.
```

Comment:
I don't understand the "area DN solid angle" part of this unit.

Reply:
Yeah I was just looking at this. I think it's the ordering that makes things confusing. It should be area DN per photon solid angle per pixel. Same with the units in the equation.